okio-fakefilesystem/build.gradle.kts

```diff
@@ -37,12 +37,19 @@ kotlin {
     configureOrCreateNativePlatforms()
   }
   sourceSets {
-    commonMain {
+    val commonMain by getting {
       dependencies {
         api(libs.kotlin.time)
         api(projects.okio)
       }
     }
+    val commonTest by getting
+    if (kmpWasmEnabled) {
```

Comment:
Insert this on line 39?

```
  if (kmpWasmEnabled) {
    configureOrCreateWasmPlatform()
  }
```

That might make it unnecessary to create the source sets here.

Reply:
Oh, I see it on line 49. Don’t mind me.